product_import/wizard/product_import_view.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="product_import_form" model="ir.ui.view">
+        <field name="name">product.import.form</field>
+        <field name="model">product.import</field>
+        <field name="arch" type="xml">
+            <form string="Import Products">
+                <group name="help-import">
+                    <div>
+                        <p
+                        >Upload below the product Catalogue as XML file. When you click on the Import button:</p>
```

Comment:
Below it says "If it is an XML file", so I guess we could support different formats there, no?
```suggestion
                        >Upload below the product Catalogue. When you click on the Import button:</p>
```

Reply:
Potentially it can support different formats. Is not the by default. But yeah, this msg should come from the specific module.